lib/glimesh/accounts/user.ex

```diff
@@ -109,6 +110,14 @@ defmodule Glimesh.Accounts.User do
     |> delete_change(:password)
   end
 
+  defp validate_displayname(changeset) do
+    username = get_field(changeset, :username)
+
+    changeset
+    |> validate_length(:displayname, min: 3, max: 50)
```

Comment:
The complexity requirements for displayname/password/username are good candidates to be pulled out into a config file or constant somewhere. Might be outside the scope for this change though. We can open an issue on it if we think it's worth addressing later.

Reply:
This should be it's own PR/issue and not a part of this one.